ci/build_linux.sh

```diff
@@ -27,7 +27,7 @@ cd temp
 if [ "$TRAVIS_PULL_REQUEST" = "false" ]
 then
 	echo "************************* Generating config files *************************" >> $BUILDLOGS
-	cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON -DBUILD_DEBUG_METRICS=ON -DPXSCENE_TEST_HTTP_CACHE=ON .. >>$BUILDLOGS 2>&1;
+	cmake -DBUILD_PX_TESTS=ON -DBUILD_WITH_PXOBJECT_TRACKING=OFF  -DBUILD_PXSCENE_STATIC_LIB=ON -DBUILD_DEBUG_METRICS=ON -DPXSCENE_TEST_HTTP_CACHE=ON .. >>$BUILDLOGS 2>&1;
```

Comment:
Are the changes in build_linux.sh needed?  BUILD_WITH_PXOBJECT_TRACKING currently defaults to OFF.